docs/content/practices/test_runners_review.md

```diff
@@ -1,64 +1,67 @@
 # Test runners
 
-Test runner is responsible for tests run and providing test result for us.
+Test runner is a test component responsible for.
+1. Preparing the test runs
+2. Providing test results
 
-`AndroidJunitRunner` — Official solution and low-level instrument. It requires a lot of effort from engineers to run
-tests on CI and make them stable.
+`AndroidJunitRunner` — The official solution and low-level instrument. It requires a lot of effort from engineers to run
+tests on the CI and make them stable.
 
-It's worth to mention — tools are getting better year by year. However, some basic functionality still doesn't work from
-the box properly.
+It's worth mentioning that UI testing tools are getting better year by year. However, some basic functionality still doesn't work out
+of the box properly.
 
 ### 1. Problems with AndroidJunitRunner:
 
-* Overcomplicated solution with clearing
+* Overcomplicated solution with state clearing
   <br>
-  _It would be good to have only one flag which does application clearing for us.
+  _It would be good to do the state clearing of the application by setting a flag.
   <br>
-  It exists, however to have scalability on CI and opportunity to use filters, you still have to
-  install `test-services.apk` and `orchestrator.apk` to each device manually_
+  It exists, however to make it scalable on the CI as well as the opportunity to use filters, you still have to
+  install `test-services.apk` and `orchestrator.apk` on each device manually_
 
 
 * Impossibility to scale
   <br>
-  _As soon as you started your tests, it's impossible to connect more devices to tests run on fly_
+  _As soon as you started your tests, it's impossible to let more devices join the test execution on the fly_
```

Comment:
I think explaining why it can help to add test on the fly and what is the case would help.

I was thinking about running test on a device cloud? There could be more available after a while? @NoNews what is the case exactly?